oeq-ts-rest-api/src/Common.ts

```diff
@@ -71,5 +71,6 @@ export interface PagedResult<T> {
  *
  * @param instance An instance to validate.
  */
-export const isPagedBaseEntity = (instance: unknown): boolean =>
-  is<PagedResult<BaseEntity>>(instance);
+export const isPagedBaseEntity = (
+  instance: unknown
+): instance is PagedResult<BaseEntity> => is<PagedResult<BaseEntity>>(instance);
```

Comment:
ensures that function is recognized as a type predicate

Reply:
OK my understanding is when `is<PagedResult<BaseEntity>>(instance)` returns true, we explicitly tell typescript that instance is of type `PagedResult<BaseEntity>`.